android/app/src/main/res/layout-land/fragment_free_roam.xml

```diff
@@ -6,7 +6,18 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context=".robot.FreeRoamFragment">
-    
+
+    <org.openbot.customview.AutoFitSurfaceView
```

Comment:
So I guess the video streaming will only work in the free roam fragment.

Reply:
Unfortunately, this is correct. The `CameraConnectionFragment` uses `TextureView` (as opposed t `SurficeView`, which is not compatible with RTSP library. Changing to SurfaceView involves a lit of changes to the main code, which I'm not very comfortable making, especially since we are planning to replace RTSP with WebRTC. We can also experiment with other RTSP libraries in the future. As I said, I tried [this](https://github.com/pedroSG94/RTSP-Server) library with intermittent success, and still have the code, if somebody is will ing to get it to work.